packages/react/src/components/Accordion/AccordionHeading.tsx

```diff
@@ -15,18 +15,7 @@ export type AccordionHeadingProps = {
  * <AccordionHeader>Header</AccordionHeader>
  */
 export const AccordionHeading = forwardRef<HTMLElement, AccordionHeadingProps>(
-  function AccordionHeading({ className, ...rest }, ref) {
-    return (
-      <Paragraph asChild size='sm'>
-        <u-summary
-          class={cl(
-            'ds-focus ds-paragraph ds-paragraph--sm ds-line-height--md', // Very TMP fix awaiting CSS modules
-            className,
-          )}
-          ref={ref}
-          {...rest}
-        />
-      </Paragraph>
-    );
+  function AccordionHeading(rest, ref) {
+    return <u-summary ref={ref} {...rest} />;
```

Comment:
```suggestion
  function AccordionHeading({ className, ...rest }, ref) {
    return <u-summary class={className} ref={ref} {...rest} />;
```
...for å fikse React sin custom element foreståelse :)

Reply:
Den er god, legger også inn kommentar om det, for framtiden